proof/refine/ARM/Ipc_R.thy

```diff
@@ -5653,21 +5654,21 @@ lemma replyRemoveTCB_sym_heap_scReplies [wp]:
          apply (clarsimp simp: isHead_to_head)
          apply (intro conjI; intro allI impI)
           apply clarsimp
-          apply (subgoal_tac "replyPrevs_of s r = Some ya")
+          apply (subgoal_tac "replyPrevs_of s rv = Some ya")
```

Comment:
Ideally there should be a `rename_tac` for the `rv` at the point where it is introduced, but more importantly also for the `ya` (you didn't introduce that one here, but these can really be very annoying in maintenance, so we should fix them wherever we see them).